doc/source/index.rst.template

```diff
@@ -34,6 +34,7 @@ programming language.
                     <h5 class="card-title">Getting started</h5>
                     <p class="card-text">New to <em>pandas</em>? Check out the getting started guides. They
                     contain an introduction to <em>pandas'</em> main concepts and links to additional tutorials.</p>
+                </div>
```

Comment:
This is just moving the button out one `div` so it can sit inside the `intro-card` element and align with respect to that